packages/flutter_web_plugins/lib/src/navigation/url_strategy.dart

```diff
@@ -5,11 +5,13 @@
 import 'dart:async';
 import 'dart:html' as html;
 import 'dart:ui' as ui;
+import 'dart:ui_web' as ui_web;
 
-import '../navigation_common/url_strategy.dart';
-import 'js_url_strategy.dart';
+import '../navigation_common/platform_location.dart';
 import 'utils.dart';
 
+export 'dart:ui_web' show urlStrategy, UrlStrategy;
```

Comment:
Do we actually need to expose the `urlStrategy` setter here? Users can (and should) get it directly from `dart:ui_web`, right?

Reply:
You're right. Removing.